### PR TITLE
Doc: Changelog for v1.7.0

### PR DESCRIPTION
**Pull request by jellypuno**
Signed-off-by: Jessielaine Punongbayan <jessielaine.punongbayan@broadcom.com>

## Proposed changes

Changelog for v1.7.0

## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [x] `npm run vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.7.0`
+
+- Disallow multiple profiles with same name but different capitalizations. Thanks @katelynienaber
+- Improvements for Optional Credentials. Thanks @crawr @jellypuno
```
Comment (IgorCATech) [inline CHANGELOG.md:8]: Can we indicate the specific improvements?

Comment (jellypuno) [reply CHANGELOG.md:8]: Do you want it in the changelog or in the readMe? It is mostly architecture.

Comment (IgorCATech) [reply CHANGELOG.md:8]: It's for the ReadMe but it would be nice to have a more precise note in Changelog, too.

Comment (zFernand0) [reply CHANGELOG.md:8]: Just going to drop this link here (hoping that one day we can do something similar) :yum:
https://www.conventionalcommits.org/en/v1.0.0/

**Pull request merged**